src/pykeen/pipeline.py

```diff
@@ -867,35 +881,39 @@ def pipeline(  # noqa: C901
         logging.debug('testing: %s', testing)
         if validation:
             logging.debug('validation: %s', validation)
-    logging.debug(f"model: {model}")
+    logging.debug(f"model: {model_instance.__class__.__name__}")
```

Comment:
`repr(model)` / `str(model)` may contain additional information (making it easier to debug).

Reply:
good point

Reply:
dine in dcad4ed5